packages/tools/devtools/devtools/src/test/types/validateDevtoolsPrevious.generated.ts

```diff
@@ -45,30 +45,6 @@ declare function use_old_InterfaceDeclaration_ContainerDevtoolsProps(
 use_old_InterfaceDeclaration_ContainerDevtoolsProps(
     get_current_InterfaceDeclaration_ContainerDevtoolsProps());
 
-/*
-* Validate forward compat by using old type in place of current type
-* If breaking change required, add in package.json under typeValidation.broken:
-* "TypeAliasDeclaration_ContainerKey": {"forwardCompat": false}
-*/
-declare function get_old_TypeAliasDeclaration_ContainerKey():
-    TypeOnly<old.ContainerKey>;
-declare function use_current_TypeAliasDeclaration_ContainerKey(
-    use: TypeOnly<current.ContainerKey>): void;
-use_current_TypeAliasDeclaration_ContainerKey(
-    get_old_TypeAliasDeclaration_ContainerKey());
```

Comment:
Is it well understood why these are disappearing from here?
I have a change that is trying to bring them back and I don't know why.